test/libsolidity/syntaxTests/iceRegressionTests/declarationUnaryTuple/declaration_delete_tuple.sol

```diff
@@ -7,4 +7,3 @@ contract C
 }
 // ----
 // TypeError: (68-69): Expression has to be an lvalue.
-// TypeError: (59-70): Unary operator delete cannot be applied to type tuple(,int_const 0)
```

Comment:
Can you explain why this was just removed?

Reply:
The first error message is now fatal.